src/db/spanner/models.rs

```diff
@@ -1341,6 +1574,8 @@ impl SpannerDb {
             let mut row = row?;
             existing.insert(row[0].take_string_value());
         }
+        self.check_quota(&user_id, &params.collection, collection_id)
```

Comment:
btw why do we `check_quota` here? seems a bit random of a location for it

Reply:
Ah, yeah, this was a mistake. Probably a bad paste.